### DISCUSSION TITLE
Boardfunctions - getText(title)

### DISCUSSION THREAD

**Discussion by EdNekebno** [EdNekebno/DGTCentaurMods#33] [Show and tell]
I have added a function to boardfunctions.py called getText. It is called passing a short title.

It returns text typed in by the user. They are first asked to remove all the pieces from the board. After this all squares represent a character, the character representations are shown on the e-Paper. By default it shows the first page of characters. You can change this by pressing the down button, and go back to the first with the up button. Placing any piece down on the board will beep and type the associated character (then pick it up and place somewhere else for the next one). The back button acts as a delete key. Pressing the tick button will make the function return the text typed.

I am a big fan of this project eventually having an app/some kind of web interface. But I think it would also be good if everything can also be done from the board itself.

**Comment by wormstein** [comment]
Nice features and very original.
Web interface is coming laster today. I hope to push it. It's just the basics, from where we can start building. It is the Flask web server with a web app.
